gnovm/cmd/gno/doc.go

```diff
@@ -28,7 +28,7 @@ func newDocCmd(io *commands.IO) *commands.Command {
 		commands.Metadata{
 			Name:       "doc",
 			ShortUsage: "doc [flags] <pkgsym>",
-			ShortHelp:  "get documentation for the specified package or symbol (type, function, method, or variable/constant).",
+			ShortHelp:  "get documentation for the specified package or symbol (type, function, method, or variable/constant)",
```

Comment:
`ShortHelp` shouldn't end with a dot. Eventually, a dot is added if `ShortHelp` is used when the command is executed with its help flag.

Reply:
I think it's better to let the point in the definition and have a more basic fprintf in command.go.

Reply:
Well, this `ShortHelp` is the only one that had a final dot, all others don't.

In addition, if we add dots at the end of `ShortHelp`, this will result in this when listing sub-commands : 

```
$ gno
USAGE
  <subcommand> [flags] [<arg>...]

Runs the gno development toolkit

SUBCOMMANDS
  mod         Manage gno.mod.
  test        Runs the tests for the specified packages.
  lint        Runs the linter for the specified packages.
  run         Runs the specified gno files.
  build       Builds the specified gno package.
  precompile  Precompiles .gno files to .go.
  clean       Removes generated files and cached data.
  repl        Starts a GnoVM REPL.
  doc         get documentation for the specified package or symbol (type, function, method, or variable/constant).
```

IMO I prefer without the final dot.

Reply:
Yep, agree 👍